### PR TITLE
Fix/gum 80 add skill

### PR DESCRIPTION
**Pull request by jol79**

### CODE REVIEW

**Review by katyakostikova — APPROVED** [review]

File: src/screens/profile/profile.tsx [hunk 1]
```diff
@@ -209,6 +221,15 @@ const ProfileScreen: React.FC = () => {
...
                 )}
+                <Heading level={HeadingLevel.H5} style={styles.containerHeader}>
+                  Skills
+                </Heading>
+                {skills.map(item => (
+                  <SkillCard key={item.id} style={styles.card} skill={item} />
+                ))}
+                {!skills?.length && (
+                  <EmptyListMessage>No skills here.</EmptyListMessage>
+                )}
```
Comment (R-Mielamud) [inline src/screens/profile/profile.tsx:235]: Add pull-to-refresh here

File: src/services/skill-api/index.ts [hunk 2]
```diff
@@ -17,11 +17,25 @@ class SkillApi {
...
+    );
+    return response.skills[0] as ISkill;
+  }
+
+  public async loadSkills() {
+    const response = await this.#http.load(
+      `${this.#apiPath}${ApiPath.SKILLS}`,
+      {
+        method: HttpMethod.GET,
+      }
```
Comment (R-Mielamud) [inline src/services/skill-api/index.ts]: This is unnecessary

File: src/services/skill-api/index.ts [hunk 3]
```diff
@@ -17,11 +17,25 @@ class SkillApi {
...
+    );
+    return response;
```
Comment (R-Mielamud) [inline src/services/skill-api/index.ts]: So is this. Just `return this.#http.....`

File: src/screens/profile/profile.tsx (hunk 1 above)
Comment (R-Mielamud) [reply src/screens/profile/profile.tsx:235]: See example on career experience, languages, okrs, opportunities pages

**Review by AleksandrTermenzhy — APPROVED** [review]

**Review by yevhenprybytok — APPROVED** [review]

**Pull request merged**